gef.py

```diff
@@ -8907,22 +8930,31 @@ def do_invoke(self, argv: List[str]) -> None:
         headers = ["Start", "End", "Offset", "Perm", "Path"]
         gef_print(Color.colorify("{:<{w}s}{:<{w}s}{:<{w}s}{:<4s} {:s}".format(*headers, w=gef.arch.ptrsize*2+3), color))
 
+        last_printed_filter = None
+
         for entry in vmmap:
-            if not argv:
+            names_filter = [f"name = '{x}'" for x in names if x in entry.path]
+            addrs_filter = [f"addr = {self.format_addr_filter(arg, addr)}" for arg, addr in addrs.items()
+                if entry.page_start <= addr < entry.page_end]
+            filters = names_filter + addrs_filter
+            filter_content = f"[{' & '.join(filters)}]"
```

Comment:
```suggestion
            filter_content = f"[{' & '.join([*names filter, *addrs_filter])}]"
```

Fine to ignore if you don't think this is cleaner.